projects/centre_console/src/relay_tx.c

```diff
@@ -0,0 +1,31 @@
+#include "relay_tx.h"
+#include "can_transmit.h"
+
+#define NUM_TX_RETRIES 5
+
```

Comment:
should be in header, no?

Reply:
yeah I think Imma need this to be in the header to be able to test it 🤔